amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -501,4 +513,99 @@ type ColonyAction @model {
   toDomain: Domain @hasOne(fields: ["toDomainId"])
   fundamentalChainId: Int
   newColonyVersion: Int
+  roles: ColonyActionRoles
+
+  # Required since some actions might have multiple event entries in the list
+  # which need to be iterated over and exposed individually
+  #
+  # This should be a list/array, but this is yet another thing Amplify is bad at
+  # The official response from them is to use DynamoDB directly:
+  # https://stackoverflow.com/questions/50992106/append-item-to-list-using-aws-appsync-to-dynamodb
+  # Which is just freaking great!
+  #
+  # As such, I've decided to simplify this, an just pass a stringified JSON around
+  # since this is for display purpouses only.
+  #
+  # If anyone has a better idea on how to handle this, I'll gladly change it!
+  #
+  # When / if you decide to use it, just be mindful of what format other people
+  # are using, and follow suit.
+  #
+  # Roles Types:
+  # [{ id: String, type: String, role: Number, setTo: boolean }]
+  individualEvents: String
```

Comment:
I think I'm suggesting the same approach as Jakub above, but with motion messages all we do is fetch the current array, append to it, and then overwrite it. 

I would prefer that approach to this (type safety, consistency)

Reply:
As I've mentioned above, I'm not particularly happy with this, and will gladly change it to something better.

Shall we set up a call ? @willm30 @jakubcolony ?